templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p>Thanks for visiting our website!</p>
+  <a href="{% url 'login' %}">Login again</a>
```

Comment:
The 'login' URL is used here. Ensure that the URL pattern for login is correctly defined in your project's URL configuration to avoid any NoReverseMatch errors.